### PR TITLE
Improve tests for rechunking

### PR DESCRIPTION
**Pull request by hendrikmakait**

### CODE REVIEW

File: tests/benchmarks/test_rechunk.py [hunk 1]
```diff
@@ -35,47 +37,73 @@ def configure_rechunking(request, memory_multiplier):
...
+    configure_rechunking_in_memory,
     small_client,
 ):
     """2D array sliced into square tiles becomes sliced by columns.
     This use case can be broken down into N independent problems.
     In task rechunk, this generates O(N) intermediate tasks and graph edges.
     """
     memory = cluster_memory(small_client)
-    shape = scaled_array_shape(memory * memory_multiplier, ("x", "x"))
+    shape = scaled_array_shape(memory * 1.5, ("x", "x"))
```
Comment (hendrikmakait) [inline tests/benchmarks/test_rechunk.py:71]: This test checks that this is still fast at a size where _the full_ dataset is too large for memory but partials should fit into memory.

File: tests/benchmarks/test_rechunk.py [hunk 2]
```diff
@@ -35,47 +37,73 @@ def configure_rechunking(request, memory_multiplier):
...
 
-    a = da.random.random(shape, chunks="auto")
+    a = da.random.random(shape, chunks=input_chunk_size)
     a = a.rechunk((-1, "auto")).sum()
     wait(a, small_client, timeout=600)
 
 
-def test_swap_axes(
+def test_swap_axes_in_memory(
```
Comment (hendrikmakait) [inline tests/benchmarks/test_rechunk.py:78]: I've split `test_swap_axes` into different tests to make it easier to read the charts.

**Comment by hendrikmakait** [comment]
Coiled estimates the cost of `test_rechunk.py` to increase from $0.12 to $0.35 per run, adding up to an increase of ~$4/month.

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx

**Pull request merged**